content/en/docs/advanced-tasks/overriding-catalog-source-pod-scheduling-configuration.md

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
Interesting - looks like the linting check is complaining about https://github.com/operator-framework/olm-docs/blob/master/content/en/docs/best-practices/channel-naming.md?plain=1#L135. We can override this check, or add another commit that fixes it, depending on what you want to do.